packages/core/src/Site/index.js

```diff
@@ -762,26 +763,23 @@ class Site {
      to trigger multiple page builds before the first one has finished building,
```

Comment:
let's move this comment to the original call site of the function since the use case is expanded

Reply:
Right, moving this to the call site at `changeCurrentPage`